### PR TITLE
Added release notes AsciidoctorJ 1.6.0

### PR DESCRIPTION
**Pull request by robertpanzer**
* [x] REMINDER: Don't forget to update the date of the post before merging.

### CODE REVIEW

**Comment by mojavelinux** [comment]
@graphitefriction a comment from @Mogztter:

> The first part about SemVer might be a bit long especially if we explain it in another post

**Comment by robertpanzer** [comment]
@mojavelinux To me this looks great! Thanks for the additions!
I am ready to make the release whenever it fits. (Maybe not tonight 😉)

File: news/asciidoctorj-1-6-0-released.adoc [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+This release brings stability and key enhancements to new and existing users alike.
+Grab a glass so we can pour you all the details.
+
+== A major stepping-stone to SemVer
+
+*Despite the version number, this release should be considered a major (breaking) release.*
+
+Asciidoctor 1.6.0 is a transitional release in preparation for making the switch to Semantic Versioning (or SemVer, for short) in 2.0.0.
+SemVer is a logical system for choosing version numbers that uses a simple MAJOR.MINOR.PATCH sequence.
+A MAJOR version is for when you make incompatible changes.
```
Comment (graphitefriction) [inline news/asciidoctorj-1-6-0-released.adoc]: The "you" in lines 35, 36, and 37, makes it feel like, me, the reader will be making these releases :wink:. I'd suggest either changing the "you" to "we" or removing the pronoun all together. For example: A MAJOR version is for introducing incompatible changes.

File: news/asciidoctorj-1-6-0-released.adoc [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+In order for AsciidoctorJ to continue moving forward, breaking changes had to be made to the API and packaging.
+Using the knowledge gained, we completely redesigned the integration to correct flaws in the original design.
+As a result, this redesign removes glitches you may have experienced when writing extensions or walking the document model with the 1.5.x releases.
+We also split the API and implementation classes into separate packages and JARs in order to isolate them.
+
+Reworked extension API::
+Thanks to the redesign, the JRuby based implementation is much better hidden.
+The interfaces for the Abstract Syntax Tree (AST) were renamed to better represent their purpose.
+For example, the AbstractNode interface was renamed to ContentNode, AbstractBlock was renamed to StructuralNode, and Inline was renamed to PhraseNode.
+And even more of the AST is now mapped in Java, including list item and table cell nodes.
```
Comment (graphitefriction) [inline news/asciidoctorj-1-6-0-released.adoc]: Each sentence on lines 68 and 69 both start with "And". Maybe have one of them start differently, just to break up the rhythm? For example: Plus, even more of the AST ...

But this is just my personal preference, so feel free to ignore this comment :smile:

File: news/asciidoctorj-1-6-0-released.adoc [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+And there are no more references to RubyObject in the API.
+Overall, the APIs for extensions and the AST should feel like a pure Java library now.
++
+Check the {url-converter-api}[extension documentation] to learn more.
+
+Added converter API::
+It's now possible to implement a converter to make your own output format entirely in Java.
+The {url-converter-api}[documentation] also shows how you get started here.
```
Comment (graphitefriction) [inline news/asciidoctorj-1-6-0-released.adoc]: Delete "also"; change "you" to "to"; delete "here".

File: news/asciidoctorj-1-6-0-released.adoc [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
++
+There's already a {url-leanpub-converter}[converter to Leanpub-flavored Markdown] implemented in Groovy that uses this API.
+
+Access to Asciidoctor's logs::
+Asciidoctor logs messages when there are errors or warnings in the documents using the Ruby logger.
+When integrating Asciidoctor into your own software, it's important to capture these log messages in order to respond to them.
+The {url-log-handling-api}[Log Handling API] will pick up messages logged in Asciidoctor and route them to your own handler.
+
+Better documentation::
+There's also an extended and tested {url-integrator-guide}[documentation] for writing extension and converters.
```
Comment (graphitefriction) [inline news/asciidoctorj-1-6-0-released.adoc]: Delete "also".

File: news/asciidoctorj-1-6-0-released.adoc [hunk 5]
```diff
@@ -0,0 +1,118 @@
...
+In most cases, migrating an existing extension is just a matter of conforming to the new API, which mostly involves updating method signatures and switching to the new names for AST interfaces.
+
+The {url-maven-plugin}[Asciidoctor Maven Plugin] and the {url-gradle-plugin}[Asciidoctor Gradle Plugin] already work with this release (primarily because they only rely on the public load/convert APIs, which have remained unchanged).
+
+Starting with 1.6.0, AsciidoctorJ will align its Java compatibility with the https://www.oracle.com/technetwork/java/java-se-support-roadmap.html[official Oracle Java SE support roadmap].
+While AsciidoctorJ doesn't require Oracle Java SE, that roadmap serves as a useful signpost for which versions of Java are considered current.
+Asciidoctor 1.6.0 will require at least Java 8.
+It also runs on, and is tested against, Java 11.
+
+Asciidoctor 1.6.0 does not yet support running on the module path yet, a feature of the {url-jpms}[Java Platform Module System] introduced in Java 9.
```
Comment (graphitefriction) [inline news/asciidoctorj-1-6-0-released.adoc]: Delete one of the "yet" 's from this sentence.

File: news/asciidoctorj-1-6-0-released.adoc [hunk 6]
```diff
@@ -0,0 +1,118 @@
...
+== Outlook for 2.0.0
+
+The breaking changes aren't over just yet.
+Heading towards 2.0.0, we want to further split the API of AsciidoctorJ and its implementation.
+Our big goal is to support alternative implementations underneath, like Asciidoctor.js, using the same public API.
+
+To help us get to 2.0.0, we ask that you test 1.6.0 and let us know if you run into any problems or if any of the changes prevent you from migrating to it.
+Now is our chance to get it right before the 2.0.0 release.
+Please file any issues you find in the https://github.com/asciidoctor/asciidoctorj/issues[issue tracker].
+Thank you for coming on this journey with us as work to bring the very best of AsciiDoc to the JVM.
```
Comment (graphitefriction) [inline news/asciidoctorj-1-6-0-released.adoc]: Sentence is missing a "we" between "as" and "work".

File: news/asciidoctorj-1-6-0-released.adoc (hunk 1 above)
Comment (graphitefriction) [reply news/asciidoctorj-1-6-0-released.adoc]: I can definitely get behind @Mogztter 's comment about trimming down the SemVer section. You could just lead with the sentence on line 33 and then link them to the SemVer blog post for more info.

**Comment by graphitefriction** [comment]
@robertpanzer, Dan told me to go ahead and make the changes and condense the semver section, so I ended up resolving my own review :laughing: :stuck_out_tongue_closed_eyes:

**Comment by robertpanzer** [comment]
@graphitefiction Thanks, this is awesome! Not being a native speaker I realize every day that I still have to learn a lot 😅
So, really a huge Thank You!!! and have a 🥳 Happy New Year!

**Comment by mojavelinux** [comment]
@robertpanzer I'd say you did a fantastic job, whether you're a non-native speaker of English or not. Most of what we added was information about Asciidoctor planning in general and how AsciidoctorJ aligns with it.

**Comment by mojavelinux** [comment]
What a way to start the new year!!

**Comment by graphitefriction** [comment]
@robertpanzer You did great! The changes I made aren't anything I wouldn't do to Dan's posts or my own.

Happy New Year to you, too! I hope it includes marvelous :beers:

**Comment by robertpanzer** [comment]
🙏 
I’ll do the release tomorrow afternoon!
Thank you everybody for your great support. This is really a great community! 
Happy new year to you! 🎉😍🍻

**Comment by mojavelinux** [comment]
That means so much to us. Thanks @robertpanzer!

**Pull request merged**